game_generator.js

```diff
@@ -6,18 +6,18 @@ module.exports = {
   newGame:  function(){
     // var board = new Board();
     var board = this.board;
-    var rows = board.length;
-    var columns = board[0].length;
+    var rows = board.rowCount();
+    var columns = board.columnCount();
     for (var r = 0; r < rows; r++) {
       $('.gameboard').append("<div class=row id=row-"+r+"></div>");
       for(var c = 0; c < columns; c++){
         $('#row-'+r).append("<div class=' square "+
-                             board[r][c].active +
+                             board.board[r][c].active +
```

Comment:
is this cleaner? Now we have to have to write board.board?  I think we would have to change some name...